src/Renci.SshNet/SftpClient.cs

```diff
@@ -37,8 +37,8 @@ public class SftpClient : BaseClient
         /// Gets or sets the operation timeout.
         /// </summary>
         /// <value>
-        /// The timeout to wait until an operation completes. The default value is negative
-        /// one (-1) milliseconds, which indicates an infinite timeout period.
+        /// The timeout to wait until an operation completes. The default value is <see cref="SshNet.Session.InfiniteTimeSpan"/>

```

Comment:
Please revert this. I do not plan on exposing **Session.InfiniteTimeSpan** publicly, and as such we should not mention it in our docs.